e2e-tests/playwright/tests/functional/channels/account_settings/profile/mm_53377_regression_test.spec.ts

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {expect, test} from '@e2e-support/test_fixture';
+import {createRandomUser} from '@e2e-support/server';
+
+test('MM-T53377 Profile popover should show correct fields after at-mention autocomplete', async ({pw, pages}) => {
+    // # Initialize with specific config and get admin client
+    const {user, adminClient, team} = await pw.initSetup();
+
+    adminClient.patchConfig({
```

Comment:
Test failed for this needs `await`
```suggestion
    await adminClient.patchConfig({
```